server/graphql/v2/object/Host.js

```diff
@@ -36,6 +38,12 @@ export const Host = new GraphQLObjectType({
           return get(collective, 'settings.tos');
         },
       },
+      plan: {
+        type: new GraphQLNonNull(HostPlan),
+        resolve(host) {
+          return host.getPlan();
+        },
+      },
```

Comment:
I had to add this to GQLV2 because the "Pay" button needs to know about the host plan.